mods/vines/glowvine.lua

```diff
@@ -0,0 +1,36 @@
+
+-- New Vines for Xen
+
+
+
+ minetest.register_node('vine:luminoustreevine', {
+	 drawtype = "plantlike",
+	 inventory_image = "dvd_luminousvineend.png",
+	 weild_image = "dvd_luminousvine.png",
+	 description = 'Twilight Vine End',
```

Comment:
When the mapgen decorator runs, there's currently no way to specify that 'vine ends' should always be placed at the bottom of vines ... so in practice this will end up being treated as a separate plant type. Probably should be named accordingly, instead of "Vine End"

Reply:
Also missing from both nodes:
`movement_speed_multiplier = default.SLOW_SPEED_PLANTS`
and
`sounds = default.node_sound_leaves_defaults()`